### PR TITLE
To Clear The Filter From Search Functionality

### PR DESCRIPTION
**Pull request by Himanshunegi0332**
Targets #2992 
### Description
Currently we don't have any feature that if a user has applied the name filter then how to remove that filter, for this I have created a functionality to remove the filter.

### Checklist:
- [X] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/CodeTrek/Resources/views/index.blade.php [hunk 1]
```diff
@@ -40,6 +40,9 @@
         </div>
         </form>
     </ul>
+    <div class="text-right fz-14 mr-11 mt-3">
+        <span>Result for 'search text'</span><a href="/codetrek"> To clear the filter click here X </a>
```
Comment (GauravGusain98) [inline Modules/CodeTrek/Resources/views/index.blade.php:44]: This `'search text'` should be replaced with the text that user has searched. If user has not search anything then the whole sentence should not be seen

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: Modules/CodeTrek/Resources/views/index.blade.php [hunk 2]
```diff
@@ -40,6 +40,11 @@
         </div>
         </form>
     </ul>
+    @if (request()->get('name'))
+    <div class="text-right fz-14 mr-11 mt-3">
+        <span>Result for '{{ request()->get('name') }}'</span><a href="/codetrek"> To clear the filter click here X </a>
```
Comment (GauravGusain98) [inline Modules/CodeTrek/Resources/views/index.blade.php:45]: The UI is not looking good. Can we make it like this. Just add a simple clear button
image

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

**Comment by GauravGusain98** [comment]
### Testing Feedbacks
- I've searched `Gaurav Gusain` but the field is only showing `Gaurav`

**Comment by Himanshunegi0332** [comment]
> ### Testing Feedbacks
> * I've searched `Gaurav Gusain` but the field is only showing `Gaurav`
> 
> image

**Review by GauravGusain98 — APPROVED** [review]
LGTM

**Pull request merged**